src/main/java/org/folio/inventory/resources/Items.java

```diff
@@ -592,12 +610,27 @@ private void addItem(
     RoutingContext routingContext,
     WebContext webContext,
     Item newItem,
-    ItemCollection itemCollection) {
+    ItemCollection itemCollection,
+    UserCollection userCollection) {
+
+    String userId = routingContext.request().getHeader("X-Okapi-User-Id");
```

Comment:
Why does this not use the method added to the context?

Reply:
Replaced by fetching the `userId` from the `WebContext`.